### PR TITLE
enable more vp9 8bit/10bit vdenc test for ffmpeg-vaapi and gstreamer

### PR DESCRIPTION
**Pull request by zhuqingliang**
this PR is based on https://github.com/intel/vaapi-fits/pull/129

### CODE REVIEW

File: lib/parameters.py [hunk 1]
```diff
@@ -338,6 +338,51 @@ def gen_vp9_vbr_parameters(spec):
   params = gen_vp9_vbr_variants(spec)
   return keys, params
 
+def gen_vp9_cqp_lp_variants(spec):
+  for case, params in spec.iteritems():
```
Comment (zhoujd) [inline lib/parameters.py]: We are working on port python2 to python3. 
For python3 style, Can you try to replace spec.iteritems() with iter(spec.items()) ?

File: lib/parameters.py [hunk 2]
```diff
@@ -338,6 +338,51 @@ def gen_vp9_vbr_parameters(spec):
...
+          variant.get("looplvl", 0), variant.get("loopshp", 0),
+        ]
+
+def gen_vp9_cbr_lp_parameters(spec):
+  keys = ("case", "gop", "bitrate", "fps", "slices", "refmode", "looplvl", "loopshp")
+  params = gen_vp9_cbr_lp_variants(spec)
+  return keys, params
+
+def gen_vp9_vbr_lp_variants(spec):
+  for case, params in spec.iteritems():
```
Comment (zhoujd) [inline lib/parameters.py]: The same as above

File: test/gst-msdk/encode/10bit/vp9.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+from ..encoder import EncoderTest
+
+spec = load_test_spec("vp9", "encode", "10bit")
+
+class VP9_10EncoderTest(EncoderTest):
+  def before(self):
+    vars(self).update(
+      codec         = "vp9",
+      gstencoder    = "msdkvp9enc",
+      gstdecoder    = "ivfparse ! msdkvp9dec hardware=true",
```
Comment (uartie) [inline test/gst-msdk/encode/10bit/vp9.py]: Please see #160... ivfparse is no longer correct.

**Comment by uartie** [comment]
@xuguangxin needs ownership please

File: test/gst-msdk/encode/10bit/vp9.py (hunk 3 above)
Comment (xhaihao) [reply test/gst-msdk/encode/10bit/vp9.py]: You may use ivfparse if you use avmux_ivf in the command line for encoding but avmux_ivf is available with the latest gst-libav, so you have to build the latest gst-libav first when running  your test cases.

**Comment by xuguangxin** [comment]
@zhangyuankun-star.

please help with this.
thanks

**Pull request closed without merging**